packages/salesforcedx-vscode-lightning/package.json

```diff
@@ -46,7 +46,7 @@
     "mocha": "3.2.0",
     "sinon": "^2.3.6",
     "typescript": "3.1.6",
-    "vscode": "1.1.17"
+    "vscode": "^1.1.36"
```

Comment:
@lcampos do we care about these deps being the same version across the different projects?